modules/search/pages/search-query-auto-complete-code.adoc

```diff
@@ -20,14 +20,26 @@ Create and configure a Search index that supports autocomplete, then configure y
 
 To add autocomplete with the Search Service to your application:
 
-. xref:search-query-auto-complete-ui.adoc#search-index[Create an autocomplete Search index with the UI] or xref:create-search-index-rest-api.adoc[] with the following JSON payload: 
+. Copy the follwoing JSON Search index payload template.
+JSON payload: 
 +
 [source,json]
 ----
 include::example$autocomplete-search-index.jsonc[]
 ----
+. Update the following strings in the template to create a new file (example index called "e-ngram-2-8" for keyspace *travel-sample.inventory.hotel* field *name*)
++
+[cols="1,1"]
+|===
+| "name": "$INDEX_NAME, | "name": "e-ngram-2-8",
+| "sourceName": "$BUCKET_NAME", | "sourceName": "travel-sample",
+| "$SCOPE_NAME.$COLLECTION_NAME": { | "inventory.hotel": {
+| "$FIELD_NAME": { | "name": {
+| "name": "$FIELD_NAME", | "name": "name",
+|===
+. xref:search-query-auto-complete-ui.adoc#search-index[Create an autocomplete Search index with the UI] or xref:create-search-index-rest-api.adoc[] with the file you created. 
 [start=2]
-. xref:simple-search-rest-api.adoc[Run a Search query] with 2-8 characters in the `query` property to test the Search index.
+. xref:simple-search-rest-api.adoc[Run a Search query] with 2-8 characters in the `query` property to test the Search index.  For the sample index "e-ngram-2-8," test the strings B, Be, Bea, Beau, and Beauf to locate the "Beaufort Hotel." Please note that the single character B is disregarded, as a minimum of two characters is required.
```

Comment:
We don't know that they're using this against the travel sample index so I'm not recommending this.

Reply:
This is such a detailed item that without a concrete example on a keyspace and a field it might fail and frustrate the user. For example if the field that is to be a autocomplete is nested three levels deep the doc text just won't work.